src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/pages/QuickAccessParametersFixtures.kt

```diff
@@ -0,0 +1,334 @@
+    val populationValueLabel
+        get() = jLabel(byXpath("//div[@text='Population']"))
+
+    // The population value spinner
+    val populationValueSpinner
+        get() = jSpinner(byXpath("//div[@class='JSpinner' and @tooltiptext='Population size of genetic algorithm.']"))
+
+    // The arrow to increase the value
+    val populationValueUpArrow
+        get() = basicArrowButton(byXpath("//div[@class='BasicArrowButton' and @name='Spinner.nextButton' and @tooltiptext='Population size of genetic algorithm.']"))
+
+    // The arrow to decrease the value
+    val populationValueDownArrow
+        get() = basicArrowButton(byXpath("//div[@class='BasicArrowButton' and @name='Spinner.previousButton' and @tooltiptext='Population size of genetic algorithm.']"))
+
+    // The text with the default value for the population value TODO: find it
+//    val populationValueDefaultTooltip
+//        get() = jLabel(byXpath(""))
+
+    /**
+     * ACTION LINK AND BUTTONS
+     */
+
+    // The action link text
+    val advancedSettingsButton
+        get() = actionLink(byXpath("//div[@class='ActionLink']"))
+
+    // The save button
+    private val saveButton
+        get() = button(byXpath("//div[@text='Save']"))
+
+    // The reset button
+    private val resetButton
+        get() = button(byXpath("//div[@text='Reset']"))
+
+    /**
+     * Clicks on the "Parameters" tab to open the panel with parameters.
+     */
+    fun openQuickAccessParametersTab() {
+        parametersTab.click()
+    }
+
+    /**
+     * Gets all the titles, namely, the title and three category separators.
+     *
+     * @return the list of titles, as described above
+     */
+    fun getTitles(): List<JLabelFixture> {
+        return listOf(
+            title, searchBudgetSeparator,
+            timeoutsSeparator, geneticAlgorithmSeparator
+        )
+    }
+
+    /**
+     * Gets the labels of the UI elements (checkboxes and spinners etc.).
+     *
+     * @return the list of labels of the UI elements
+     */
+    fun getUIElementLabels(): List<JLabelFixture> {
+        return listOf(
+            searchBudgetTypeLabel, searchBudgetValueLabel, initializationTimeoutLabel,
+            minimisationTimeoutLabel, assertionTimeoutLabel, jUnitCheckTimeoutLabel,
+            populationLimitLabel, populationValueLabel
+        )
+    }
+
+    /**
+     * Gets save and reset buttons in one list.
+     *
+     * @return the list with save and reset buttons
+     */
+    fun getButtons(): List<JButtonFixture> {
+        return listOf(saveButton, resetButton)
+    }
+
+    /**
+     * Gets the tooltips with default values of the parameters.
+     *
+     * @return the list of "default tooltips"
+     */
+    fun getDefaultTooltips(): List<JLabelFixture> {
+        // TODO: pass others once they are found
+        return listOf(
+            searchBudgetTypeDefaultTooltip,
+            initializationTimeoutDefaultTooltip,
+            populationLimitDefaultTooltip
+        )
+    }
```

Comment:
It appears that this method is not used anywhere in the project?

Reply:
Have removed it

Reply:
Good